app/bundles/MessengerBundle/DependencyInjection/EnvProcessor/MessengerNullableEnvVarProcessor.php

```diff
@@ -6,6 +6,9 @@
 
 use Symfony\Component\DependencyInjection\EnvVarProcessorInterface;
 
+/**
+ * @deprecated since Mautic 5.0, to be removed in 6.0 with no replacement.
+ */
```

Comment:
```suggestion
```
This is being used https://github.com/mautic/mautic/blob/a33b41b32a2bce629b15c6098a71da071a080a4c/app/config/config.php#L95